src/header/map.rs

```diff
@@ -14,11 +14,48 @@ use super::HeaderValue;
 pub use self::as_header_name::AsHeaderName;
 pub use self::into_header_name::IntoHeaderName;
 
-/// A set of HTTP headers
+/// The `HeaderMap` type is a specialized
```

Comment:
Keep a vert short sentence to start helps when the type is listed on a module page, such as: https://docs.rs/http/latest/http/header/index.html